osu.Game/Beatmaps/Drawables/Cards/HoverHandlingContainer.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Input.Events;
+
+namespace osu.Game.Beatmaps.Drawables.Cards
+{
+    public class HoverHandlingContainer : Container
```

Comment:
Is this going to be used elsewhere? If not I'd just inline it in `BeatmapCardExtraInfoRow`.

Reply:
`BeatmapCardContent` (in the new naming) is also using that in 2 places. Generally I would not have bothered with having this be a separate class if I didn't need it in 3 places, because it gets rather verbose if I do it the conventional way by overriding `OnHover(Lost)`.

Reply:
Ah, guess I missed it because it's non-abstract usage.